.travis-build.sh

```diff
@@ -7,6 +7,15 @@ bootstrap_header() {
   echo -e "${ANSI_YELLOW}$1${ANSI_RESET}"
 }
 
+travis_fold() {
```

Comment:
That looks like a great idea... Will try that, but it could be a few days until I can come back to this... I also would like to have a solution for the zmw problem on OSX first...